core/App/ehsm_napi.cpp

```diff
@@ -1125,4 +1128,64 @@ char* NAPI_ExportDataKey(const char* cmk_base64,
     return retJsonObj.toChar();
 }
 
-}  // extern "C"
+
+    char *NAPI_RA_HANDSHAKE_MSG0(const char *p_msg0)
```

Comment:
keep the same indend with the above funtions

Reply:
It will be fix.